src/StagedProposalProcessorSetup.sol

```diff
@@ -37,55 +46,68 @@ contract StagedProposalProcessorSetup is PluginUpgradeableSetup {
     /// to verify the plugin on the respective block explorers.
     constructor() PluginUpgradeableSetup(address(new SPP())) {}
 
-    /// @notice The ID of the permission required to call the `storeNumber` function.
-    bytes32 internal constant STORE_PERMISSION_ID = keccak256("STORE_PERMISSION");
-
     /// @inheritdoc IPluginSetup
     function prepareInstallation(
         address _dao,
         bytes calldata _data
     ) external returns (address plugin, PreparedSetupData memory preparedSetupData) {
```

Comment:
Avoid confusion with the inner plugins

```suggestion
    ) external returns (address spp, PreparedSetupData memory preparedSetupData) {
```

Reply:
done in https://github.com/aragon/staged-proposal-processor-plugin/pull/8/commits/9cb984c4cbe559a051943fde540da25b853c2400